core-bundle/src/Image/PictureFactory.php

```diff
@@ -168,22 +178,29 @@ private function createConfig($size): array
                 if (null !== $imageSizes) {
                     $options->setSkipIfDimensionsMatch((bool) $imageSizes->skipIfDimensionsMatch);
 
-                    $config->setFormats(array_merge(
-                        [],
-                        ...array_map(
-                            static function ($formatsString) {
-                                $formats = [];
-
-                                foreach (explode(';', $formatsString) as $format) {
-                                    [$source, $targets] = explode(':', $format, 2);
-                                    $formats[$source] = explode(',', $targets);
-                                }
-
-                                return $formats;
-                            },
-                            StringUtil::deserialize($imageSizes->formats, true)
-                        )
-                    ));
+                    $formatsString = implode(';', StringUtil::deserialize($imageSizes->formats, true));
+                    $formats = [];
+
+                    foreach (explode(';', $formatsString) as $format) {
```

Comment:
If you remove the `implode(';'` above, you do not have to use `explode(';'` here, do you?

Reply:
I do, the strings themself can include `;` like the JPEG one does.

Reply:
I see. 👍